src/nnet3bin/nnet3-egs-augment-image.cc

```diff
@@ -28,6 +28,8 @@
 namespace kaldi {
 namespace nnet3 {
 
+enum fill_mode_type { kNearest, kReflect};
```

Comment:
According to Google style, enum name should be in first-letter-capital format: i.e. FillModeType or simply FillMode.